pytest-backend/exasol/pytest_backend/version.py

```diff
@@ -6,5 +6,5 @@
 # If you need to change the version, do so in the project.toml, e.g. by using `poetry version X.Y.Z`.
 MAJOR = 0
 MINOR = 2
-PATCH = 1
+PATCH = 2
```

Comment:
This is actually a minor version jump, because we add a new feature. Patch is only for bugfixes